### PR TITLE
feat: vault sharing backend

### PR DESCRIPTION
**Pull request by HEARC-Miguel-Moreira**
Django's vault sharing backend's implementation

### CODE REVIEW

File: api/firstpassapp/urls.py [hunk 1]
```diff
@@ -15,5 +16,9 @@
          views.get_vaults_for_user, name='vaults-for-user'),
     path('vaults/new/', views.create_vault, name='create-vault'),
     path('images/save/', views.save_image, name='save-image'),
+    path('invitations/user/<int:user_id>/', views.get_invitations_for_user, name='invitations-for-user'),
+    path('invitations/accept/<int:invitation_id>/', views.accept_invitation, name='accept-invitation'),
```
Comment (Xstoudi) [inline api/firstpassapp/urls.py]: `/invitations/<id>/accept` plutôt je pense

**Review by TimothyPeck — APPROVED** [review]

File: api/firstpassapp/urls.py [hunk 2]
```diff
@@ -15,5 +16,9 @@
          views.get_vaults_for_user, name='vaults-for-user'),
     path('vaults/new/', views.create_vault, name='create-vault'),
     path('images/save/', views.save_image, name='save-image'),
+    path('invitations/user/<int:user_id>/', views.get_invitations_for_user, name='invitations-for-user'),
+    path('invitations/accept/<int:invitation_id>/', views.accept_invitation, name='accept-invitation'),
+    path('invitations/decline/<int:invitation_id>/', views.decline_invitation, name='decline-invitation'),
```
Comment (Xstoudi) [inline api/firstpassapp/urls.py]: `/invitations/<id>/decline` plutôt

File: api/firstpassapp/urls.py [hunk 3]
```diff
@@ -15,5 +16,9 @@
          views.get_vaults_for_user, name='vaults-for-user'),
     path('vaults/new/', views.create_vault, name='create-vault'),
     path('images/save/', views.save_image, name='save-image'),
+    path('invitations/user/<int:user_id>/', views.get_invitations_for_user, name='invitations-for-user'),
```
Comment (Xstoudi) [inline api/firstpassapp/urls.py]: `users/<id>/invitations` je pense

File: api/firstpassapp/urls.py [hunk 4]
```diff
@@ -15,5 +16,9 @@
          views.get_vaults_for_user, name='vaults-for-user'),
     path('vaults/new/', views.create_vault, name='create-vault'),
     path('images/save/', views.save_image, name='save-image'),
+    path('invitations/user/<int:user_id>/', views.get_invitations_for_user, name='invitations-for-user'),
+    path('invitations/accept/<int:invitation_id>/', views.accept_invitation, name='accept-invitation'),
+    path('invitations/decline/<int:invitation_id>/', views.decline_invitation, name='decline-invitation'),
+    path('users/vault/<int:vault_id>/', views.get_users_for_vault, name='users-for-vault'),
```
Comment (Xstoudi) [inline api/firstpassapp/urls.py]: `vaults/<id>/users` =)

**Review by Xstoudi — CHANGES_REQUESTED** [review]

**Pull request merged**